### DISCUSSION TITLE
IMDB watch list plugin broken?

### DISCUSSION THREAD

**Discussion by thedavix12** [Flexget/Flexget#4001] [Q&A]
Hi,

I'm using the IMDB watch list since years now (didn't change any config since I can remember). 

However, since a month now, the task doesn't run anymore and I have the following error
`imdb_watchlist  There was an error during imdb_watchlist input (Unable to get imdb list from imdb react widget. Either the list is empty or the imdb parser of the imdb_watchlist plugin is broken. Original error: 'NoneType' object has no attribute 'group'.), using cache instead.`

I tried to revert flexget to version 3.11.29 when I didn't have the problem, but it doesn't seems to be related to the version, as I still get this error.
Maybe something changed on the imdb watchlist page and the parse is broken?

Thanks

**Comment by chewy74** [comment]
https://github.com/Flexget/Flexget/issues/3998
